src/api/app/views/webui2/webui/package/_fileinfo.html.haml

```diff
@@ -0,0 +1,38 @@
+%dl.row.mt-4
+  %dt.col-md-2.text-truncate Title
+  %dd.col-md-10= fileinfo.value(:summary)
```

Comment:
jfyi: value() is converting the argument .to_s - so using strings is actually a little faster. But it really doesn't matter in views.

Reply:
@coolo, thanks! I've just opened a new PR for this: #7420